Screenbox.Core/Enums/SettingsOptions.cs

```diff
@@ -20,3 +20,15 @@ public enum ThemeOption
     Light,
     Dark
 }
+
+public enum LaunchPageOption
+{
+    Home,
+    Songs,
+    Albums,
+    Artists,
+    VideoFolders,
+    AllVideos,
+    Network,
+    PlayQueue
+}
```

Comment:
Having the launch page name as an enum is also problematic in the future. If we add new page types to the app, we have to add the page at the bottom of the enum. The order of this enum is fixed once it is released. We cannot change this order (only append to it) because doing so will break the user's settings.

Reply:
This is true, but we could also hardcode the enum values, then we could insert new ones in the middle. Though that would not be all that elegant either.